[Bug]: Chromium webdriver URL for version 116.x not found

### Describe the Bug

Unit-tests for saving altair png and svg caught invalid Chromium webdriver endpoint url with error 404 response.

Fix is to update wedriver_manager version to 4.0.0

### Instructions to Reproduce the Bug

Run pytest after clean project/ package install.

### Traceback

```shell
_____________________________ test_save_png_altair _____________________________

    def test_save_png_altair():
        """Test that that figures saved as png exist."""
        fig = alt.Chart(pd.DataFrame()).mark_bar()
        path = "tests/temp/"
>       saving.save(fig, "test_fig", path)

tests/viz/altair/test_saving.py:13: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
nesta_ds_utils/viz/altair/saving.py:115: in save
    driver = _google_chrome_driver_setup() if driver is None else driver
nesta_ds_utils/viz/altair/saving.py:26: in _google_chrome_driver_setup
    ChromeDriverManager().install(), chrome_options=chrome_options
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/chrome.py:39: in install
    driver_path = self._get_driver_path(self.driver)
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path
    file = self._download_manager.download_file(driver.get_url())
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/download_manager.py:28: in download_file
    response = self._http_client.get(url)
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:33: in get
    self.validate_response(resp)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

resp = <Response [404]>

    @staticmethod
    def validate_response(resp: requests.Response):
        status_code = resp.status_code
        if status_code == 404:
>           raise ValueError(f"There is no such driver by url {resp.url}")
E           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/116.0.5845/chromedriver_linux64.zip

/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:16: ValueError
_____________________________ test_save_svg_altair _____________________________

    def test_save_svg_altair():
        """Test that that figures saved as svg exist."""
        fig = alt.Chart(pd.DataFrame()).mark_bar()
        path = "tests/temp/"
>       saving.save(fig, "test_fig", path, save_png=False, save_svg=True)

tests/viz/altair/test_saving.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
nesta_ds_utils/viz/altair/saving.py:115: in save
    driver = _google_chrome_driver_setup() if driver is None else driver
nesta_ds_utils/viz/altair/saving.py:26: in _google_chrome_driver_setup
    ChromeDriverManager().install(), chrome_options=chrome_options
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/chrome.py:39: in install
    driver_path = self._get_driver_path(self.driver)
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path
    file = self._download_manager.download_file(driver.get_url())
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/download_manager.py:28: in download_file
    response = self._http_client.get(url)
/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/webdriver_manager/core/http.py:33: in get
    self.validate_response(resp)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

resp = <Response [404]>

    @staticmethod
    def validate_response(resp: requests.Response):
        status_code = resp.status_code
        if status_code == 404:
>           raise ValueError(f"There is no such driver by url {resp.url}")
E           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/116.0.5845/chromedriver_linux64.zip
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -14,7 +14,7 @@ python_requires = >=3.8
 install_requires =
     numpy==1.23.4
     pandas==1.5.1
-    pyyaml==6.0
+    pyyaml<5.4.0
     scipy==1.9.3
     pyarrow==10.0.0
 [options.extras_require]

--- a/setup.cfg
+++ b/setup.cfg
@@ -25,7 +25,7 @@ viz =
   altair-saver==0.5.0
   matplotlib==3.6.2
   selenium==4.2.0
-  webdriver_manager==3.8.4
+  webdriver_manager==4.0.0
 networks =
   networkx==2.8.8
 nlp =

